init:
  joint_names: [ 'Body' ]
  time_from_start: 2.0
  positions: [ 0.0, 0.0, 1.39, 0.34, -1.39, -1.04, 0.0, 0.0, 0.0, 0.0, -0.43, 0.69, -0.34, 0.0, 0.0, 0.0, -0.43, 0.69, -0.34, 0.0, 1.39, -0.34, 1.39, 1.04, 0.0, 0.0 ]
zero:  
  joint_names: [ 'Body' ]
  time_from_start: 2.0
  positions: [ 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0 ] 
crouch:
  joint_names: [ 'Body' ]
  time_from_start: 2.0
  positions: [ 0.0, 0.0, 1.545, 0.33, -1.57, -0.486, 0.0, 0.0, -0.3, 0.057, -0.744, 2.192, -1.122, -0.035,-0.3, 0.057, -0.744, 2.192, -1.122, -0.035, 1.545, -0.33, 1.57, 0.486, 0.0, 0.0 ]
sit:
  joint_names: [ 'Body' ]
  time_from_start: 2.0
  positions: [ 0.0, 0.179436, 0.931096, 0.157960, -0.564554, -1.259372, -0.026120, 1.0, -0.892746, 0.308376, -1.583046, 1.391296, 0.849794, -0.009162, -0.892746, -0.332836, -1.561654, 1.356098, 0.862150, 0.023052, 0.917374, -0.041460, 0.559868, 1.001744, 0.631966, 1.0 ]
headInitial:
  joint_names: [ 'HeadYaw', 'HeadPitch' ]
  time_from_start: 1.0
  positions: [ 0.0, 0.179436 ]
headDown:
  joint_names: [ 'HeadYaw', 'HeadPitch' ]
  time_from_start: 1.5
  positions: [ 0.0, 0.514872 ]
sitWithHeadDown:
  joint_names: [ 'Body' ]
  time_from_start: 2.0
  positions: [ 0.0, 0.514872, 0.931096, 0.157960, -0.564554, -1.259372, -0.026120, 1.0, -0.892746, 0.308376, -1.583046, 1.391296, 0.849794, -0.009162, -0.892746, -0.332836, -1.561654, 1.356098, 0.862150, 0.023052, 0.917374, -0.041460, 0.559868, 1.001744, 0.631966, 1.0 ]
laneOne:
  joint_names: [ 'HeadYaw', 'HeadPitch', 'LShoulderPitch', 'LShoulderRoll', 'LElbowYaw', 'LElbowRoll', 'LWristYaw', 'LHand' ]
  time_from_start: 1.5
  positions: [ 0.265340, 0.510780, 0.575208, 0.406468, -0.247016, -0.375788, -0.036858, 1.0 ]
laneTwo:
  joint_names: [ 'HeadYaw', 'HeadPitch', 'LShoulderPitch', 'LShoulderRoll', 'LElbowYaw', 'LElbowRoll', 'LWristYaw', 'LHand' ]
  time_from_start: 1.5
  positions: [ 0.204064, 0.511780, 0.472430, 0.004560, 0.256136, -0.168698, -0.765508, 1.0 ]
laneThree:
  joint_names: [ 'HeadYaw', 'HeadPitch', 'RShoulderPitch', 'RShoulderRoll', 'RElbowYaw', 'RElbowRoll', 'RWristYaw', 'RHand' ]
  time_from_start: 2
  positions: [ 0.0, 0.510780, 0.515466, 0.007628, -0.291502, 0.428028, 1.257838, 1.0 ]
laneFour:
  joint_names: [ 'HeadYaw', 'HeadPitch', 'RShoulderPitch', 'RShoulderRoll', 'RElbowYaw', 'RElbowRoll', 'RWristYaw', 'RHand' ]
  time_from_start: 2
  positions: [ -0.156510, 0.511780, 0.467912, -0.173384, -0.420358, 0.431096, 0.908086, 1.0 ] 
laneFive:
  joint_names: [ 'HeadYaw', 'HeadPitch', 'RShoulderPitch', 'RShoulderRoll', 'RElbowYaw', 'RElbowRoll', 'RWristYaw', 'RHand' ]
  time_from_start: 2
  positions: [ -0.286900, 0.510780, 0.429562, -0.388144, -0.457174, 0.391212, 0.503110, 1.0 ]   